### PR TITLE
Scrollable performance refactor

### PR DESCRIPTION
**Pull request by yairEO**
* Updated to version `1.5.9`
* Updated `Scrollable` documentation
* Improved `CustomScrollbars` example page

### CODE REVIEW

File: src/content/examples/CustomScrollbars/style.scss [hunk 1]
```diff
@@ -1,32 +1,39 @@
...
+    border-radius: 4px;
+    width: 150px;
+    height: 150px;
+    font-size: 0.8em;
+
+    .scrollbar-inner {
+      padding: 4px 8px;
+    }
+
+    table {
```
Comment (ykadosh) [inline src/content/examples/CustomScrollbars/style.scss:24]: I think that this is a leftover from your tests maybe?

File: src/content/examples/CustomScrollbars/style.scss [hunk 2]
```diff
@@ -1,32 +1,39 @@
...
-
-      .scrollbar-inner {
-        padding: 4px 8px;
-      }
+  display: grid;
+  place-items: center;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  box-sizing: border-box;
+  padding: 20vh 30%;
```
Comment (ykadosh) [inline src/content/examples/CustomScrollbars/style.scss:7]: This makes them too far apart in my opinion:
Screen Shot 2021-11-10 at 10 29 00

File: src/content/examples/CustomScrollbars/index.jsx [hunk 3]
```diff
@@ -46,8 +47,8 @@ const Scrollbar4 = () => {
     );
 };
 
-const Scrollbars = ({children}) => (
-    <Scrollable>
+const Scrollbars = ({children, ...props}) => (
+    <Scrollable {...props}>
```
Comment (ykadosh) [inline src/content/examples/CustomScrollbars/index.jsx:51]: I don't see you passing anything other than children anywhere

File: src/content/examples/CustomScrollbars/style.scss (hunk 1 above)
Comment (yairEO) [reply src/content/examples/CustomScrollbars/style.scss:24]: Yes :D good catch! I've created a crazy stress-test full of DOM nodes directly in the example page and then removed it

File: src/content/examples/CustomScrollbars/index.jsx (hunk 3 above)
Comment (yairEO) [reply src/content/examples/CustomScrollbars/index.jsx:51]: yeah I passed the new prop (`cssVarsOnTracks`) in the last example but then decided to remove it. I can remove the spread also

File: src/content/examples/CustomScrollbars/style.scss (hunk 2 above)
Comment (yairEO) [reply src/content/examples/CustomScrollbars/style.scss:7]: Ok I will make them closer

**Pull request merged**